suite-common/wallet-core/src/stake/stakeThunks.ts

```diff
@@ -59,12 +60,12 @@ export const fetchEverstakeData = createThunk<
 
 export const initStakeDataThunk = createThunk(
     `${STAKE_MODULE}/initStakeDataThunk`,
-    (_, { getState, dispatch, extra }) => {
-        const enabledNetworks = extra.selectors.selectEnabledNetworks(getState());
+    (_, { getState, dispatch }) => {
+        const networks = selectDeviceNetworkSymbolsOfVisibleAccounts(getState());
```

Comment:
I not sure if this is good solution. On desktop this relies on enabled networks instead of accounts, so if you will have blank desktop app, connect device, enable eth it will take another 2 minutes to fetch stake data because this thunk will not run again when new account is added.

On mobile it's not issue because mobile initiate this fetch when user enters staking detail screen.

Reply:
👍  You are right. I didnt realize that desktop does not refetch it when entering staking. I'll think about it once more.

Reply:
a108320107